draft-ion.adoc

```diff
@@ -105,16 +120,13 @@ A single Ion instance resource without metadata is simply a standard JSON object
 }
 ----
 
-A single Ion instance resource with metadata is a JSON object that has a direct
-child `meta` name/value pair in addition to its other member name/value pairs:
+An Ion _Resource_ is a JSON object with a Metadata member named `meta` in addition to one or more other members (name/value pairs):
```

Comment:
Zero or more additional members is just called an 'Ion Object':

> Ion Object
> A JSON object with a Meta Object member named meta.

As for this point:

> a resource that is a link to some other resource for whatever reason

This is currently defined as an Ion Link:

> Ion Link
> An Ion Object where its meta member contains an IRI member named href. 

An Ion Link could have only a Meta Object, but that Ion Link is not considered an Ion Resource.

The definition of `Resource` has the additional burden of having one or more members beyond its `meta` member.

I rationalize it like this:

There is no such thing as metadata that doesn't correspond to data (i.e. the very definition of metadata is 'data that describes other data', 'other' being the key word).  This means either that 'other' data has to be local (peer members of the `meta` object) or available in some other location, at which point a link is required.  Without either of these two conditions, it can't be considered Metadata by the definition of the word (or by Ion's definition) and an Ion parser should just treat it as normal JSON.

An Object without a `meta` member is still a valid JSON object, but it just doesn't have any Ion semantic meaning.  This means that, in order to be recognized by an Ion parser, a Resource *must* have **one** or more members _in addition to_ is `meta` member, otherwise the object is treated as normal JSON.

It might also help to visualize this as a taxonomy tree:

              Ion Object
          ________|________
          |               |
    Ion Link         Ion Resource    
                          |
                Ion Collection Resource

A Resource can simultaneously be considered a Link - and vice versa - if the JSON structure has a `meta` `href` and one or more members peer to the `meta` member.

In summary, these definitions (Ion Resource, Ion Link, etc) are *only* to indicate the JSON structures that Ion treats as having special semantic meaning.  If they don't match semantic criteria, they still exist, they're just treated as normal JSON, not Ion concepts.

Does this make sense?